examples/firebase/index.js

```diff
@@ -0,0 +1,179 @@
+const functions = require("firebase-functions");
+const { PKPass } = require("passkit-generator");
+const admin = require("firebase-admin");
+var fs = require('file-system');
+var path = require('path');
+var axios = require('axios');
+var os = require('os');
+
+// Firebase init
+admin.initializeApp({
+    credential: admin.credential.cert(require("CERTIFICATE_PATH")),
+    storageBucket: "STORAGE_BUCKET_URL"
+});
+var storageRef = admin.storage().bucket()
+
+exports.pass = functions.https.onRequest((request, response) => {
+    PKPass.from({
+        // Get relevant pass model from model folder (see passkit-generator/examples/models/)
+        model: `./model/${request.body.passType}.pass`,
+        certificates: {
+            // Assigning certificates from certs folder (you will need to provide these yourself)
+            wwdr: fs.fs.readFileSync("./certs/wwdr.pem"),
+            signerCert: fs.fs.readFileSync("./certs/signerCert.pem"),
+            signerKey: fs.fs.readFileSync("./certs/signerKey.pem"),
+            signerKeyPassphrase: "PASSPHRASE"
+        }, 
+    },
+        {
+            authenticationToken: "AUTH_TOKEN",
+            webServiceURL: "https://us-central1-YOUR-FUNCTION.cloudfunctions.net/FUNCTION_NAME",
```

Comment:
Is there a Firebase API that returns you, on runtime, the URL of the cloud function?

Perhaps this might be useless, as well as `authenticationToken`...

Reply:
Yes these are probably both useless, looking back it looks like I just copied and pasted the function URL from the Firebase console and generated my own random auth token, so they can probably be removed!

Reply:
Yeah, remove them. I'll add them back later if I'll ever setup an example cloud function that supports passes update.